Dockerfile

```diff
@@ -18,27 +21,28 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update \
 
 # setup phenoplier
 COPY environment/environment.yml environment/scripts/install_other_packages.sh environment/scripts/install_r_packages.r /tmp/
-RUN conda env create --name phenoplier --file /tmp/environment.yml \
-  && conda run -n phenoplier --no-capture-output /bin/bash /tmp/install_other_packages.sh \
+RUN conda env create --name ${CONDA_ENV_NAME} --file /tmp/environment.yml \
+  && conda run -n ${CONDA_ENV_NAME} --no-capture-output /bin/bash /tmp/install_other_packages.sh \
   && conda clean --all --yes
```

Comment:
I've used docker on occasion, but I know very little about the details. Is there a reason to make all of this a single `RUN` command instead of three different lines? Does docker clear environment variables between `RUN` commands or is this just personal preference?